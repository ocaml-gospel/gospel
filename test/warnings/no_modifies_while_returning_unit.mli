val no_modifies_while_returning_unit : int -> unit
(*@ no_modifies_while_returning_unit i
    requires true *)
