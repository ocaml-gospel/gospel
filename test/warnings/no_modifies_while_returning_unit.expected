File "no_modifies_while_returning_unit.mli", line 1, characters 0-120:
Warning
The function `no_modifies_while_returning_unit' returns a unit but its specifications does not contains any `modifies' clause.
